app/src/main/java/com/example/mystore/ui/shared/CustomTopBar.kt

```diff
@@ -42,7 +44,8 @@ fun CustomTopBar(
         },
         navigationIcon = {
             if (canGoBack)
-                IconButton(onClick = { destinationsNavigator.navigateUp() }) {
+                IconButton(onClick = { destinationsNavigator.navigateUp() })
+                {
```

Comment:
Not sure why there's been a change here but the curly brace belongs on the upper row in Java/Kotlin, as it was before the change.